governance/xc_admin/packages/xc_admin_frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-hot-toast": "^2.4.0",
+    "sharp": "^0.33.4",
```

Comment:
why is this needed

Reply:
Sharp should be installed  for any production environment running nextjs (outside e.g.g Vercel where it is already part of the environment) and is used for image optimizations, but [it's actually mandatory for standalone mode](https://nextjs.org/docs/messages/sharp-missing-in-production) which the new `Dockerfile` uses to significantly reduce the size of the resulting image.